genet/output/sanitiser.py

```diff
@@ -14,11 +14,10 @@ def sanitise_list(x):
 def sanitise_geodataframe(gdf):
     if isinstance(gdf, GeoSeries):
         gdf = GeoDataFrame(gdf)
-    gdf = gdf.fillna("None")
     object_columns = gdf.select_dtypes(["object"]).columns
     for col in object_columns:
         if gdf[col].apply(lambda x: isinstance(x, (set, list))).any():
-            gdf[col] = gdf[col].apply(lambda x: ",".join(x))
+            gdf[col] = gdf[col].apply(lambda x: ",".join(x) if isinstance(x, (list, set)) else x)
```

Comment:
How is this handled when loading in data into genet that has previously gone through genet's sanitiser?

Reply:
we're not reading networks from geojson/etc files